src/main/java/com/suse/saltstack/netapi/client/SaltStackClient.java

```diff
@@ -385,4 +386,19 @@ public Job call() throws SaltStackException {
         };
         return executor.submit(callable);
     }
+
+  public Stats stats() throws SaltStackException {
+      return connectionFactory.create("/stats", JsonParser.STATS, config).getResult();
+  }
+
+  public Future<Stats> statsAsync() {
+      Callable<Stats> callable = new Callable<Stats>() {
+          @Override
+          public Stats call() throws SaltStackException {
+              return stats();
+          }
+      };
+      return executor.submit(callable);
+  }
+
```

Comment:
Sometimes you add this empty line, sometimes not. It would be better to stay consistent with the rest of the project. Sorry for the nitpick. :)